HttpFs.UnitTests/RequestBody.fs

```diff
@@ -213,19 +213,19 @@ let bodyFormatting =
             NameValue ("user_name", "Åsa den Röde")
             NameValue ("user_pass", "Bović")
         ]
-        |> fun form -> Impl.formatBody clientState (None, utf8, BodyForm form)
-        |> fun (newCt, body) ->
-          let bodyToString = body |> bodyToBytes |> utf8.GetString
-          Expect.equal bodyToString "submit=Join+Now!&user_name=%C3%85sa+den+R%C3%B6de&user_pass=Bovi%C4%87" "body should be equal"
+        |> fun form -> Impl.createHttpContent clientState (None, utf8, BodyForm form)
+        |> fun (newCt, message) ->
+          let bodyToString = message.ReadAsStringAsync() |> Async.AwaitTask |> Async.RunSynchronously
```

Comment:
Never use `RunSynchronously`, instead use the `testAsync` stuff https://github.com/haf/expecto#normal-tests